packages/flame/lib/src/game/base_game.dart

```diff
@@ -242,14 +247,22 @@ class BaseGame extends Game with FPSCounter {
   }
 
   @override
-  Vector2 projectOffset(Offset value) {
-    final vector = value.toVector2();
-    return camera.screenToWorld(vector);
+  Vector2 projectVector(Vector2 vector) {
+    return _combinedProjector.projectVector(vector);
+  }
+
+  @override
+  Vector2 unprojectVector(Vector2 vector) {
+    return _combinedProjector.unprojectVector(vector);
+  }
+
+  @override
+  Vector2 projectDelta(Vector2 vector) {
```

Comment:
The rename of this method don't make much sense to me. Velocity isn't a delta. The previous name "scaleOffset" worked better IMO.

I am ok with changing them to Vector2 instead of Offset, but I would rethink the name, maybe just scaleVector?

Reply:
> Velocity isn't a delta

of course it is haha. delta s over delta t. what it isn't is is an offset. that is why I changed `scaleOffset`. also I think it's better for the user to know which one to use if we call it by what it's used for, not what it does. the fact that it scales and doesn't translate is an implementation detail, imo